### ISSUE TITLE
Arch linux install/usage instructions not correct

### ISSUE THREAD

**Issue by morganm** [certbot/certbot#3141]
The [documentation page](https://certbot.eff.org/#arch-apache) for ArchLinux appears to be incorrect or outdated. The first step says to install the 'letsencrypt' package, but this doesn't exist in the Arch or AUR repositories:

```
> pacman -Ss letsencrypt
> packer -Ss letsencrypt
aur/letsencrypt.sh-git 0.2.0.r4.gd5b2858-1 (4)
    A Let's Encrypt (ACME protocol) client shell script
aur/letsencrypt-systemd-nginx 1.0-2 (2)
    Systemd service for Let's Encrypt certificate renewal with nginx servers
aur/ca-certificates-letsencrypt 20160325-1 (1)
    Let's Encrypt CA certificates
aur/letsencrypt-cli 0.3.0-1 (0)
    Yet another Letsencrypt (ACME) client using Ruby.
```

Several other matching options do show up in AUR. But as it stands, the instructions written on the doc page do not work so a first time user would have to some searching and then trial and error experimentation with these various clients, instead of having a straightforward documented user experience.

I did find a certbot package that is related to letsencrypt, although it is a different command binary than what listed in the doc page ("certbot" instead of "letsencrypt"). The command usage seems similar, though. Not sure if this should be the official client listed in the documentation instead?

**Comment by tido-dot-com** [comment]
[Operating System Packages](https://certbot.eff.org/docs/using.html#operating-system-packages) has the same issue

The letsencrypt packages have all been renamed to certbot* and now certbot is the correct command as documented in [The name of the certbot command](https://certbot.eff.org/docs/using.html#operating-system-packages)

```
$ pacman -Ss certbot
community/certbot 0.8.0-1 [installed]
    A tool to automatically receive and install X.509 certificates to enable TLS
    on servers. The client will interoperate with the Let’s Encrypt CA which
    will be issuing browser-trusted certificates for free.
community/certbot-apache 0.8.0-1
    Apache plugin for Let’s Encrypt client
community/certbot-nginx 0.8.0-1 [installed]
    Nginx plugin for Let’s Encrypt client
community/letshelp-certbot 0.6.0-2
    Let's help Let's Encrypt client
```

**Comment by tido-dot-com** [comment]
Additionally it may be worth referencing the official Arch Linux Let’s Encrypt (and certbot) documentation:
https://wiki.archlinux.org/index.php/Let%E2%80%99s_Encrypt

**Comment by bmw** [comment]
> The documentation page for ArchLinux appears to be incorrect or outdated.

I have an issue (certbot/website#128) and PR (certbot/website#129) against the website repo to fix this. Arch shipped packages with the new names in the last few days and this change should be fixed to the website shortly.

> Operating System Packages has the same issue

This has been fixed in [master](https://github.com/certbot/certbot/blob/master/docs/using.rst#operating-system-packages). It will also be fixed on the website shortly.

> Additionally it may be worth referencing the official Arch Linux Let’s Encrypt (and certbot) documentation

That's a good idea. I created certbot/website#131 to track that issue.

Since all of these issues are against the website, I'm closing in favor of issues on the website repo. Regardless, thanks for reporting these problems guys.
